### PR TITLE
Symfony 5.4

### PR DESCRIPTION
**Pull request by rimi-itk**
Upgrades to Symfony 5.4 security.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,32 +114,32 @@ security:
...
+In order to use the CLI login feature the following environment variable must be
+set:
 
 ```shell
 DEFAULT_URI=
 ```
 
-See [Symfony documentation](https://symfony.com/doc/current/routing.html#generating-urls-in-commands)
+See [Symfony
+documentation](https://symfony.com/doc/current/routing.html#generating-urls-in-commands)
```
Comment (turegjorup) [inline README.md]: Why line break here?

File: src/Security/OpenIdLoginAuthenticator.php [hunk 2]
```diff
@@ -47,7 +47,7 @@ public function supports(Request $request)
     /**
      * @throws ValidationException
      */
-    public function getCredentials(Request $request)
+    protected function getCredentials(Request $request)
```
Comment (turegjorup) [inline src/Security/OpenIdLoginAuthenticator.php]: I would prefer a rename to `getClaims` as you suggested on Teams

**Review by turegjorup — APPROVED** [review]
Approved with comments.

File: src/Security/OpenIdLoginAuthenticator.php (hunk 2 above)
Comment (rimi-itk) [reply src/Security/OpenIdLoginAuthenticator.php]: Fixed in https://github.com/itk-dev/openid-connect-bundle/pull/12/commits/c4ad38b489660ded6cca2d13e8f2ca030c1a83be

File: README.md (hunk 1 above)
Comment (rimi-itk) [reply README.md]: `fill-paragraph` (cf. https://www.gnu.org/software/emacs/manual/html_node/emacs/Fill-Commands.html) doesn't handle lines containing (long) urls very well. Line break removed in https://github.com/itk-dev/openid-connect-bundle/pull/12/commits/682dbe2268ab2d59db6fd3ef52da299d31f727b6.

**Review by turegjorup — APPROVED** [review]

**Pull request merged**